concepts/05 Widgets/CheckBox/10 Handle the Value Change Event.md

```diff
@@ -39,6 +39,63 @@ To process a new **CheckBox** value, you need to handle the value change event.
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxCheckBox
+            :on-value-changed="onValueChanged"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxCheckBox from 'devextreme-vue/check-box';
+
+    export default {
+        components: {
+            DxCheckBox
+        },
+        data() {
+            return {
+                onValueChanged: function (e) {
+                    let previousValue = e.previousValue;
+                    let newValue = e.value;
+                    // Event handling commands go here
+                }
+            };
+        },
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import CheckBox from 'devextreme-react/check-box';
+
+    class App extends React.Component {
+        checkBox_valueChanged(e) {
```

Comment:
Why is this function called differently in different approaches? Please standardize. I suggest removing the excessive information, so that the name will be like onValueChanged or handleValueChange

Reply:
fixed